hack/run-functests.sh

```diff
@@ -21,6 +22,12 @@ if [ -n "${IMAGE_FORMAT}" ]; then
   LATENCY_TEST_RUN="true"
 fi
 
+if [[ -z "${CNF_TEST_IMAGE}" ]]; then
```

Comment:
I see 2 occurrences of the env var:
- CNF_TEST_IMAGE
- CNF_TEST**S**_IMAGE
Which one is the right one ?

Reply:
the code consumes `CNF_TESTS_IMAGE` (with the S)